pipelines/graphene_nightly_pipeline

```diff
@@ -23,18 +23,18 @@ pipeline {
                     }
                 }
 
-				stage ('Invoke Ubuntu 18.04 with 5.19 Graphene native Suite') {
-                    steps {
-                        build job: 'local_ci_graphene_native_18.04_5.19', propagate: true, wait: true
-                    }
-                }
-
-                stage ('Invoke Ubuntu 18.04 with 5.19 Graphene SGX Suite') {
-                    steps {
-                        sleep(time: 15, unit: "MINUTES")
-                        build job: 'local_ci_graphene_sgx_18.04_5.19', propagate: true, wait: true
-                    }
-                }
+				// stage ('Invoke Ubuntu 18.04 with 5.19 Graphene native Suite') {
```

Comment:
can you use block comment /* */ instead of line comment? Will be easier to read and revert back. Also add a comment to explain that this is temporary as the machine is used by Jitender to run some attestation experiments.